src/routes.php

```diff
@@ -40,4 +40,7 @@
     $router->get('projects/{id}/images/filter/annotation-label/{id2}', [
         'uses' => 'ProjectImageAnnotationLabelController@index',
     ]);
+    $router->group(['namespace' => 'Geojson', 'prefix' => 'geojson'], function($router){
+      $router->get('volume/{id}/images', ['uses' => 'VolumeImagesController@index']);
```

Comment:
The route should be `volumes`, as model identifiers in routes are always plural in BIIGLE.

```suggestion
      $router->get('volumes/{id}/images', ['uses' => 'VolumeImagesController@index']);
```